aips/create_dips_job.py

```diff
@@ -219,6 +243,86 @@ def filter_aips(aips, location_uuid):
         help="Set the debugging output level. This will override -q and -v",
     )
 
+    # Delete argument can't be set in the two subparsers bellow with the same name
+    parser.add_argument(
+        "--delete-local-copy",
+        action="store_true",
+        help="Deletes the local DIPs after upload if any of the upload arguments is used.",
+    )
+
+    # Create optional upload type subparsers
+    subparsers = parser.add_subparsers(
+        dest="upload_type",
+        title="Upload options",
+        description="The following arguments allow to upload the DIP after creation:",
+        help="Leave empty to keep the DIP in the output path.",
+    )
+
+    # Storage Service upload subparser with extra SS required arguments
+    parser_ss = subparsers.add_parser(
+        "ss-upload",
+        help="Storage Service upload. Check 'create_dips_job ss-upload -h'.",
+    )
+    parser_ss.add_argument(
+        "--pipeline-uuid",
+        metavar="UUID",
+        required=True,
+        help="UUID of the Archivemativa pipeline in the Storage Service",
+    )
+    parser_ss.add_argument(
+        "--cp-location-uuid",
+        metavar="UUID",
+        required=True,
+        help="UUID of the pipeline's Currently Processing location in the Storage Service",
+    )
+    parser_ss.add_argument(
+        "--ds-location-uuid",
+        metavar="UUID",
+        required=True,
+        help="UUID of the pipeline's DIP storage location in the Storage Service",
+    )
+    parser_ss.add_argument(
+        "--shared-directory",
+        metavar="PATH",
+        help="Absolute path to the pipeline's shared directory.",
+        default="/var/archivematica/sharedDirectory/",
+    )
+
+    # AtoM upload subparser with AtoM required arguments
+    parser_atom = subparsers.add_parser(
+        "atom-upload", help="AtoM upload. Check 'create_dips_job atom-upload -h'."
+    )
+    parser_atom.add_argument(
+        "--atom-url",
+        metavar="URL",
+        help="AtoM instance URL. Default: http://192.168.168.193",
+        default="http://192.168.168.193",
+    )
+    parser_atom.add_argument(
+        "--atom-email",
+        metavar="EMAIL",
+        required=True,
+        help="Email of the AtoM user to authenticate as.",
+    )
+    parser_atom.add_argument(
+        "--atom-password",
+        metavar="PASSWORD",
+        required=True,
+        help="Password of the AtoM user.",
+    )
+    parser_atom.add_argument(
+        "--atom-slug",
+        metavar="SLUG",
+        required=True,
+        help="AtoM archival description slug to target the upload.",
+    )
+    parser_atom.add_argument(
+        "--rsync-target",
+        metavar="HOST:PATH",
+        help="Destination value passed to Rsync. Default: 192.168.168.193:/tmp.",
+        default="192.168.168.193:/tmp",
```

Comment:
Is having this default going to be useful?

Reply:
Not really, I'll make this and the AtoM URL required arguments. Thanks!